### PR TITLE
Licensing

### PR DESCRIPTION
**Pull request by frederikcoppens**
This license is based on The Turing Way
Data > CC-BY
Infrastructure > MIT

Any objections on these licenses for the DM toolkit ?

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# LICENSE
+
+The ELIXIR Data Management toolkit makes all of their materials publicly available under open source licenses.
+
+The process documents and data are made available under a CC-BY license.
+Software are made available under an MIT license.
+
+The  license text listed below (describing both CC-BY and MIT licenses as well as their usage in the ELIXIR Data Management toolkit) is re-used under a CC-BY license from The Turing Way community materials.
+(Specifically from the [The Turing Way](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE.md)).
```
Comment (nsoranzo) [inline LICENSE:9]:
```suggestion
```

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Process documents and data
+
+All documentation and chapter materials in this repository are made available under the [Creative Commons Attribution license][cc-by-human].
```
Comment (nsoranzo) [inline LICENSE:13]:
```suggestion
All documentation and chapter materials in this repository
is licensed under a [Creative Commons Attribution 4.0 International License][cc-by-human].
```

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+* **Attribution**---You must give appropriate credit (mentioning that your work is derived from work that is Copyright © The Turing Way Community and, where practical, linking to https://github.com/alan-turing-institute/the-turing-way), provide a [link to the license][cc-by-human], and indicate if changes were made.
+  You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
+* **No additional restrictions**---You may not apply legal terms or technological measures that legally restrict others from doing
+anything the license permits.
+
+With the understanding that:
+
+* You do not have to comply with the license for elements of the material in the public domain or where your use is permitted by an applicable exception or limitation.
+* No warranties are given. The license may not give you all of the permissions necessary for your intended use. For example, other rights such as publicity, privacy, or moral rights may limit how you use the material.
```
Comment (nsoranzo) [inline LICENSE:35]:
```suggestion
```

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+> EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+> MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+> NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+> LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+> OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+> WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+[cc-by-human]: https://creativecommons.org/licenses/by/4.0/
+[cc-by-legal]: https://creativecommons.org/licenses/by/4.0/legalcode
```
Comment (nsoranzo) [inline LICENSE:61]:
```suggestion
```

**Comment by nsoranzo** [comment]
Duplicate of https://github.com/elixir-europe/rdm-toolkit/pull/1 ?

**Comment by bedroesb** [comment]
@frederikcoppens time to merge?

**Comment by bedroesb** [comment]
I was a bit to quick here

**Pull request merged**